package.json

```diff
@@ -8,11 +8,9 @@
   ],
   "scripts": {
     "precommit": "lint-staged && jest && flow check",
-    "lint:md-partial": "remark -u remark-preset-davidtheclark --frail --quiet --no-stdout --output --",
```

Comment:
Is Prettier going to automatically add the table of contents to the README? It's not, right?

I'd rather stick with remark for that reason (and I prefer the formatting over the changes above). We could use the CLI in remark-preset-davidtheclark directly, to remove the explicit remark-cli dependency, if we wanted to.

Reply:
Correct, it does not generate the table of contents (I didn't know remark did either).

Reason I removed remark was because I thought it was unneeded since prettier can handle the formatting instead and is already used.

Should I add remark back and remove markdown files from prettier in this PR or fix the javascript formatting issues mentioned above?

Reply:
Markdown formatting has been removed from the PR